tests/utils.go

```diff
@@ -578,7 +566,21 @@ func GenerateRESTReport() error {
 	return nil
 }
 
-func AfterTestSuitCleanup() {
+// CreateDirectory creates the a directory on the given path if it doesn't exist.
+func CreateDirectory(artifactsDir string) (string, error) {
```

Comment:
I think this functuion name should be `CreateArtifactsDirectory`

Reply:
Why? It can create an arbitrary directory and it is in "utils.go"

Reply:
Maybe change `artifactsDir` to `path`